registration/models.py

```diff
@@ -121,6 +121,8 @@ class Gender(models.TextChoices):
                     ' employee.')
     )
 
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Just for curiosity, do we also need a `updated_at` field here? This could give a hint, when the profile was verified.

Reply:
I thought about that too. The created_at field is used to easily access new profiles that are not verified yet using the admin site.  On the other hand, after verification, a user is only allowed to edit his/her phone number. These changes don't need a further verification. Other changes need to be made by a staff directly using the admin site. So if we use an `updated_at` field to sort by, profiles at the top that are not at the top of `created_at` would be profiles that either have a new phone number and don't need verification or profiles that were changed by a staff and are already verified. This is why I decided against an `updated_at` field.